src/interfaces/breadcrumb-interface.ts

```diff
@@ -0,0 +1,9 @@
+import Vue from 'vue'
+
+export interface BreadcrumbInterface extends Vue {
```

Comment:
It is Breadcrumb or BreadCrumb? Code uses both :)

Reply:
I believe it's one word. Let me search out those camelcase instances.